docs/release-notes.md

```diff
@@ -16,6 +16,8 @@ nav_order: 9
 ### Changes
 
 - Warn if template for enabled systemd instance unit has no `Install` section
+- Warn if filesystem device is the same as the disk device
+- Warn if `wipeFilesystem` is set to false and `wipeTable` is set to true
```

Comment:
These might be clearer if they described the consequence we're trying to catch, rather than the mechanics of the tests.